src/main/java/aquality/selenium/core/configurations/TimeoutConfiguration.java

```diff
@@ -0,0 +1,58 @@
+package aquality.selenium.core.configurations;
+
+import aquality.selenium.core.utilities.ISettingsFile;
+import com.google.inject.Inject;
+
+public class TimeoutConfiguration implements ITimeoutConfiguration{
+
+    private final ISettingsFile settingsFile;
+    private final long condition;
+    private final long pollInterval;
+    private final long implicit;
+    private final long command;
+
+    @Inject
+    public TimeoutConfiguration(ISettingsFile settingsFile) {
+        this.settingsFile = settingsFile;
+        condition = getTimeout(TIMEOUT.CONDITION);
+        pollInterval = getTimeout(TIMEOUT.POLL_INTERVAL);
+        implicit = getTimeout(TIMEOUT.IMPLICIT);
+        command = getTimeout(TIMEOUT.COMMAND);
+    }
+
+    private long getTimeout(TIMEOUT timeout){
+        return Long.valueOf(settingsFile.getValue("/timeouts/" + timeout.getKey()).toString());
```

Comment:
would not it be better to move "/timeouts/" in the enumeration to keep full jsonpath in the one place? and getKey can be renamed to getPath or something like that

Reply:
agree